<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Moscow Cars Dealership</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addRow">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Create account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletRow">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Delete account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLine">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GO!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sort by:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Price: Low to High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Price: High to Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mileage: Low to High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mileage: High to Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year: Oldest to Newest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year: Newest to Oldest</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QLineEdit" name="yearLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameLine">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="9" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="capacityLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Engine Capacity</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="seatsLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="14" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="priceLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Price (million ruble)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="ownerlLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Owner type</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="transmissionLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="powerLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="fullNameLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="locationLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="locationLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="mileageLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="yearLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLineEdit" name="priceLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="capacityLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="6" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="fuelLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Fuel Type</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="transmissionLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Transmission</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="powerLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Power</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="seatsLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Seats</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="mileageLabel">
          <property name="font">
           <font>
            <family>Reem Kufi</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Mileage (km)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="fuelLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="ownerLine">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
